index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
-    var i = { size: 4, blocks: [0x0F00, 0x2222, 0x00F0, 0x4444], color: 'purple'   };
-    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'pink'   };
```

Comment:
Hello, I think the problem might be coming from using close colors. I tried it on my machine using different colors and the error dissappeared. Perhaps you could try that and see if the error goes away.